### DISCUSSION TITLE
Model Viewer is capable to load multiple models (5-7) on webpage ?

### DISCUSSION THREAD

**Discussion by ti-hardikshah** [google/model-viewer#3074] [Q&A]
### Description

Hello, I want to know <Model-Viewer> is capable to load more than 5 models on a webpage?
Is there any performance issue due to load 5-6 model in Loading = "Auto"
 
### Version
- model-viewer: 1.9.2

### Browser Affected

All mobile browser 

### AR
- [x] WebXR
- [x] SceneViewer
- [x] QuickLook

**Comment by ti-hardikshah** [comment]
Facing issue: (Mobile) Browser crash or refresh page automatically

**Comment by elalish** [comment]
MV doesn't limit  how many models you can load on a page. However, your GPU will, as you'll eventually run out of GPU RAM, and that doesn't tend to be handled very gracefully by the browser. You'll  probably find that you can load different numbers of models on different machines. Your best bet is  to look at  compressing your models so they don't take up so  much space. I'd highly recommend using KTX2 texture compression  which reduces GPU RAM usage considerably. You may also  consider if  you really need that many models visible at the same time; if not you may want to release their resources to free up memory.
